modules/microshift-config-parameters-table.adoc

```diff
@@ -129,7 +150,7 @@ Not all MIME types benefit from compression, but `HAProxy` uses resources to try
 
 |`ingress.logEmptyRequests`
 |`Log` or `Ignore`
-|Default value is `Log`. Specifies how connections on which empty requests are received are logged. These connections typically come from the health probes of a load balancer service health or a web browser's speculative connections, such as a `preconnect`. Logging typical requests might be undesirable, but requests can also be caused by network errors or port scans, in which case logging can be useful for diagnosing errors and detecting intrusion attempts.
+|The default value is `Log`. Specifies how connections on which empty requests are received are logged. These connections typically come from the health probes of a load balancer service health or a web browser's speculative connections, such as a `preconnect`. Logging typical requests might be undesirable, but requests can also be caused by network errors or port scans, in which case logging can be useful for diagnosing errors and detecting intrusion attempts.
```

Comment:
There are other instances, one in this PR above, where "Default" vs. "The default" is used. If you're correcting it here, should it be make consistent throughout the doc?

"health probes of a load balancer service health" <-- drop the second use of "health".